app/models/meeting.rb

```diff
@@ -3,6 +3,8 @@ class Meeting < ActiveRecord::Base
   include Listable
   include Invitable
 
+  self.ignored_columns = %w(invites_sent)
```

Comment:
@gnclmorais this was introduced in rails 5.1.7 - we are not quite there yet :)

Reply:
Good catch! For a moment I was wondering if I got the syntax wrong. 😀 Removing then… It was just a safeguard if we could.